### ISSUE TITLE
PDF renders incorrectly - Page12

### ISSUE THREAD

**Issue by blackmambahk** [mozilla/pdf.js#10145]
We have a custom viewer built on PDFJS.
The attached pdf renders incorrectly.

Attach (recommended) or Link to PDF file here:
[Test.pdf](https://github.com/mozilla/pdf.js/files/2461202/Test.pdf)

Not my pdf but from a customer, looks like it was created with Adobe InDesign and they are using some effects.
 
Take a look at Page 12 it's messed up, images are completely wrong size, way too small, and some missing, I think there was supposed to be some sort of image drop shadow effect.

View it on chrome with their native pdfium viewer and it's ok, you can also use foxit's web renderer and that's fine as well, no surprise there I guess.

FWIW and probably isn't the customer sent a screenshot of the InDesign settings for effects on that page
<img width="664" alt="fx on page 12" src="https://user-images.githubusercontent.com/947206/46682529-ca68f880-cbbb-11e8-905b-96443f61c867.png">

It's wrong on the latests FireFox and also the latest PDFJS build we use in our viewer.

**Comment by THausherr** [comment]
At pages 1, 6 and 14 there is also a weird 45° shape on the 3rd card.

**Comment by blackmambahk** [comment]
yeah I have the screenshot for the settings page for those , just went with the most obvious issue 😀

**Comment by ShumailAhmed123** [comment]
Yes, on the page no 1 and 12 on the Mozilla Firefox browser, the pdf file is acting weird. On page no 1 it totally change the image and on the page no 12 after the heading the content on the page is very small and its all on the top left hand side.
